wormchain/app/helpers/mock.go

```diff
@@ -0,0 +1,50 @@
+package helpers
+
+import (
```

Comment:
I like the way you've structured this, similar comment here though -- can we rename this file to `mock_key.go` or `mock_private_key.go` since we may add similar mock files in the future? 

Might also be a good idea to move the `helpers` folder entirely under `apptesting`, unless you were imagining that these would be used in non-testing code somewhere.

Reply:
Yes, renamed and moved to apptesting to consolidate the testing packages.